homeassistant/components/gree/switch.py

```diff
@@ -12,6 +16,29 @@
 from .const import COORDINATORS, DISPATCH_DEVICE_DISCOVERED, DISPATCHERS, DOMAIN
 from .entity import GreeEntity
 
+GREE_SWITCHES: tuple[SwitchEntityDescription, ...] = (
+    SwitchEntityDescription(
+        icon="mdi:lightbulb",
+        name="Panel light",
+        key="panel_light",
+    ),
+    SwitchEntityDescription(
+        name="Quiet",
+        key="quiet",
+    ),
+    SwitchEntityDescription(
+        name="Fresh air",
```

Comment:
The name is part of unique id. It cannot be changed.
```suggestion
        name="Fresh Air",
```